testdata/memcached-molecule-operator/config/prometheus/monitor.yaml

```diff
@@ -11,11 +11,8 @@ metadata:
 spec:
   endpoints:
     - path: /metrics
-      port: https
-      scheme: https
-      bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
-      tlsConfig:
-        insecureSkipVerify: true
+      port: http # Ensure this is the name of the port that exposes HTTP metrics
```

Comment:
So by default this is not secured?

Reply:
I believe this is an intentional change because of the kube-rbac-proxy no longer being bundled. @joelanford can you confirm?